README.md

```diff
@@ -1 +1,55 @@
-# Hello_word
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello_word
+
+> One paragraph statement about the project.
```

Comment:
[OPTIONAL] For future purposes, when using the Microverse template, you are required to customize it, which means in places where you see instructions like: "One paragraph statement about the project", it should actually be a paragraph saying something about the project.